### DISCUSSION TITLE
Why is wallpaper and style up in the pop-up menu

### DISCUSSION THREAD

**Discussion by SeanDylan** [LawnchairLauncher/lawnchair#3751] [Q&A] [answered]
In the latest debug version i saw wallpaper and style on top instead of bottom why is this?

**Comment by MrSluffy** [comment, marked as answer]
It's better, and maybe in future we can imitate like pixel launcher to switch wallpaper easily, who knows 🔢 
![3fcdb187-edea-40ba-a77a-8d2786e162c5](https://github.com/LawnchairLauncher/lawnchair/assets/36076410/a3b13b06-7f8b-4458-94ef-08952622ac99)

**Reply by SeanDylan** [reply]
Should be instead have the quick-wallpaper-picker enabled by default and this feature should make sense
